spec/Cabin/Seat.vspec

```diff
@@ -70,6 +82,25 @@ MassageLevel:
   unit: percent
   description: Seat massage level. 0 = off. 100 = max massage.
 
+MassageStatus:
+  datatype: string
+  type: sensor
+  allowed: ['ON', 'OFF']
+  description: Massage status.  
+
+SupportedMassageTypes:
+  datatype: string[]
+  type: attribute
+  allowed:  ['Off','Roll','Knead','Anti Fatigue','Pulse','Stretch','Lumbar','Shoulder','Wave','Relief','Glide']
```

Comment:
At least for me it would be good to have a short description of each term. Or is it up to OEM to define the meaning of them?

Also - no need for OFF here?